src/client/js/error.js

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const genericErrorMessage = 'Something went wrong. Please try again or come back later.'
+const ErrorActionTypes = {
+  None: 'none',
+  Toast: 'toast'
+}
```

Comment:
Mentioned briefly (and probably poorly) in previous comment, but should this be exported so other file(s) can use the error type consts?